src/components/User/Users/components/UserDetails/UserDetails.Block.vue

```diff
@@ -143,7 +144,10 @@ export default {
 
   methods: {
     async blockUser () {
-      if (!await confirmAction('Are you sure?')) {
+      if (!await confirmAction({
```

Comment:
Validate using 'confirmAction' function. Again, in all project.

Reply:
done